Configuration/StandardSequences/python/Eras.py

```diff
@@ -59,7 +59,8 @@ def __init__(self):
                            'tracker_apv_vfp30_2016', 'pf_badHcalMitigation', 'run2_miniAOD_80XLegacy','run2_miniAOD_94XFall17', 'run2_nanoAOD_92X',
                            'run2_nanoAOD_94XMiniAODv1', 'run2_nanoAOD_94XMiniAODv2', 'run2_nanoAOD_94X2016',
                            'run2_miniAOD_devel', 'run2_nanoAOD_102Xv1',
-                           'hcalHardcodeConditions', 'hcalSkipPacker']
+                           'hcalHardcodeConditions', 'hcalSkipPacker',
+                           'BParking']
```

Comment:
all the modifiers that we have so far start with a lower case.
Please keep this consistent.

Reply:
Done, [here](
https://github.com/cms-sw/cmssw/pull/25455/commits/4f7b8488fce9719cab38173cfcb6832d81157b9f)